lib/internal/http2/compat.js

```diff
@@ -417,7 +417,11 @@ class Http2ServerResponse extends Stream {
   write(chunk, encoding, cb) {
     var stream = this[kStream];
     if (stream === undefined) {
-      cb(new Error('HTTP/2 Stream has been closed'));
+      var err = new Error('HTTP/2 Stream has been closed');
+      if (cb)
```

Comment:
@mcollina According to docs, all three arguments are optional. Perhaps do something like:

```js
cb = cb || encoding || chunk;
if (typeof cb === 'function') { // ... }
```

Or more correctly:

```js
cb = arguments[arguments.length - 1];
if (typeof cb === 'function') { // ... }
```